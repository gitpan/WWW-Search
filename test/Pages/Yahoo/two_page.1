<html><head><title>Yahoo! Search Results</title></head><body><table border=0 width="100%" cellspacing=0 cellpadding=0><tr><td valign=middle width="1%">
<a href="http://www.yahoo.com"><img border=0 height=31 width=147 src="http://a1.g.a.yimg.com/7/1/31/000/us.yimg.com/i/yahoo.gif" alt="Yahoo!"></a>
</td><td valign=middle><table border=0 cellspacing=0 cellpadding=0 width="100%"><tr><td valign=bottom align=left><font face=arial size="-1"><a href="http://edit.my.yahoo.com/config/edit_searches?.ss=replication&.sn=replication&.src=s&.done=http://my.yahoo.com">Personalize</a></font></td><td align=right valign=bottom>
<font face=arial size=-1><a href="http://help.yahoo.com/help/search/">Help</a> - <a href=http://mail.yahoo.com>Check Email</a>
</font></td></tr></table><hr size=1 noshade></td></tr></table><table border=0 cellpadding=3 cellspacing=2 width="100%"><tr>
<td bgcolor="a0b8c8">
<font face=arial><b>Search Result</b></font>&nbsp;&nbsp;<font size=-1 face=arial>Found <b>0</b>
categories
and <b>173</b> 
sites
for <b>replication</b></font></td></tr></table><p>

<center>
<!-- SpaceID=0 robot -->
</center>

<p><table bgcolor="black" border=0 cellpadding=0 cellspacing=0 width="100%"><tr><td><table border=0 cellpadding=2 cellspacing=1 width="100%"><tr align=center bgcolor="eeeeee">
<td width="20%"><font size=-1 face=arial>
Categories
</font></td><td bgcolor="#a0b8c8" width="20%"><font size=-1 face=arial><b>Web Sites</b></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/top/*http://ink.yahoo.com/bin/query?p=replication&hc=0&hs=173">Web Pages</a></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/top/*http://search.news.yahoo.com/search/news?&n=84&p=replication&&hc=0&hc=0&hs=173">Related News</a></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/top/*http://search.events.yahoo.com/search/events?p=replication&hc=0&hs=173">Net Events</a></font></td></tr></table></td></tr></table><p>
<p>
<table border=0 cellpadding=0 cellspacing=4 align=right><tr><td>
<table bgcolor=black border=0 cellspacing=0><tr><td>
<table border=0 cellpadding=3 cellspacing=0>
</table></td></tr></table>
</td></tr></table>

<p>
<CENTER><FONT SIZE="+1"><B>Yahoo! Site Matches &nbsp; <FONT SIZE="-1">(85&nbsp;-&nbsp;104&nbsp;of&nbsp;173)</FONT></B></FONT></CENTER>
<p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Communications_and_Networking/Software/Networking/New_Era_of_Networks/">Business and Economy > Companies > Computers > Communications and Networking > Software > Networking > New Era of Networks </A>
<UL TYPE=disc><LI><A HREF="http://www.neonsoft.com/">Home</A>
 - message-oriented middleware facilitating data connectivity across networks and Internet, providing message queuing, dynamic formatting, rules engine, and data <b>replication</b>.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Communications_and_Networking/Software/Networking/">Business and Economy > Companies > Computers > Communications and Networking > Software > Networking </A>
<UL TYPE=disc><LI><A HREF="http://www.octopustech.com/">Octopus Technologies, Inc.</A>
 - software for Windows NT networks  providing real-time data fault tolerance, file mirroring  and data <b>replication</b> across lans and wans. </UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Communications_and_Media_Services/Duplication/Audio/">Business and Economy > Companies > Communications and Media Services > Duplication > Audio </A>
<UL TYPE=disc><LI><A HREF="http://www.maxdisk.com/">MaxDisk</A>
 - offers CD, cassette, and vinyl <b>replication</b> services.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Communications_and_Media_Services/Duplication/Computer_Media/">Business and Economy > Companies > Communications and Media Services > Duplication > Computer Media </A>
<UL TYPE=disc><LI><A HREF="http://www.softpack.com/">Software Packaging Associates</A>
 - Diskette Duplication, CD-ROM <b>Replication</b>, Direct Digital Printing, Order Fulfillment, Design Services and other services to handle the Distribution effort of your software business.<LI><A HREF="http://www.softpack.com/">Software Publishing Associates</A>
 - Diskette Duplication, CD-ROM <b>Replication</b>, Direct Digital Printing, Order Fulfillment, Design Services and other services to handle the Distribution effort of your software business.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Hardware/Peripherals/Retailers/">Business and Economy > Companies > Computers > Hardware > Peripherals > Retailers </A>
<UL TYPE=disc><LI><A HREF="http://www.datadisc.com/">dataDisc CD-ROM Tools and Technology</A>
 - provides mastering, <b>replication</b>, CDR, and document publishing services; sells CD software, recorders, blank discs, jukeboxes and  complete turnkey services.  </UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Services/Consulting/">Business and Economy > Companies > Computers > Services > Consulting </A>
<UL TYPE=disc><LI><A HREF="http://www.mburetta.com/">Marie Buretta, Inc.</A>
 - a consulting firm specializing in data distribution and <b>replication</b>, OLAP, data warehousing, and database performance and tuning.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Software/Business/Document_Management/Services/">Business and Economy > Companies > Computers > Software > Business > Document Management > Services </A>
<UL TYPE=disc><LI><A HREF="http://www.edp.com">Electronic Demand Publishing</A>
 - focuses on short-run length printing, media <b>replication</b>, fulfillment, and assembly.</UL><p>
<A HREF="http://dir.yahoo.com/Business_and_Economy/Companies/Computers/Software/Data_Conversion/">Business and Economy > Companies > Computers > Software > Data Conversion </A>
<UL TYPE=disc><LI><A HREF="http://www.scribesoft.com">Scribe Software Corporation</A>
 - developer of KitchenSync, a data integration toolkit for developing migration, conversion, and <b>replication</b> solutions between applications, databases, and legacy systems.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/New_York/Cities/Rochester/Business_and_Shopping/Business_to_Business/Computers/">Regional > U.S. States > New York > Cities > Rochester > Business and Shopping > Business to Business > Computers </A>
<UL TYPE=disc><LI><A HREF="http://www.edp.com">Electronic Demand Publishing</A>
 - focuses on short-run length printing, media <b>replication</b>, fulfillment, and assembly.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Ohio/Cities/Athens/Education/College_and_University/Public/Ohio_University/Faculty/">Regional > U.S. States > Ohio > Cities > Athens > Education > College and University > Public > Ohio University > Faculty </A>
<UL TYPE=disc><LI><A HREF="http://oak.cats.ohiou.edu/~biegalke/">Biegalke, Bonita J.</A>
 - Assistant Professor, Department of Biological Sciences. Studies the <b>replication</b> of cytomegalovirus, a herpes virus.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Pennsylvania/Cities/Exton/Business_and_Shopping/Business_to_Business/Communications_and_Media_Services/">Regional > U.S. States > Pennsylvania > Cities > Exton > Business and Shopping > Business to Business > Communications and Media Services </A>
<UL TYPE=disc><LI><A HREF="http://www.gomrs.com">Multimedia Recording Systems</A>
 - multimedia software <b>replication</b> equipment and supplies for CD-R and diskettes.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Pennsylvania/Cities/Paoli/Business_and_Shopping/Business_to_Business/Computers/">Regional > U.S. States > Pennsylvania > Cities > Paoli > Business and Shopping > Business to Business > Computers </A>
<UL TYPE=disc><LI><A HREF="http://www.iti-pa.com">ITI Shadowbase</A>
 - provides total database <b>replication</b> solutions.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Texas/Cities/Richardson/Business_and_Shopping/Business_to_Business/Communications_and_Media_Services/">Regional > U.S. States > Texas > Cities > Richardson > Business and Shopping > Business to Business > Communications and Media Services </A>
<UL TYPE=disc><LI><A HREF="http://www.merlinss.com/">Merlin Software Services</A>
 - customer driven <b>replication</b> of CD-R, CD-ROM, and floppy disks. Turnkey manufacturing, warehousing and fulfillment for any industry.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Ohio/Cities/Mentor/Business_and_Shopping/Business_to_Business/Computers/">Regional > U.S. States > Ohio > Cities > Mentor > Business and Shopping > Business to Business > Computers </A>
<UL TYPE=disc><LI><A HREF="http://www.cdrinteractive.com">CDR Interactive Media</A>
 - offers CD-ROM development and <b>replication</b> services.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/Countries/Canada/Provinces_and_Territories/Ontario/Counties_and_Regions/Peel/Cities/Mississauga/Business_and_Shopping/Business_to_Business/">Regional > Countries > Canada > Provinces and Territories > Ontario > Counties and Regions > Peel > Cities > Mississauga > Business and Shopping > Business to Business </A>
<UL TYPE=disc><LI><A HREF="http://www.cdrom4u.com">Pinnacle Computers</A>
 - CDR duplication and CD-ROM <b>replication</b> services; specializes in fast turn-around projects.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/Countries/Australia/Business_and_Economy/Companies/Communications_and_Media_Services/Duplication/">Regional > Countries > Australia > Business and Economy > Companies > Communications and Media Services > Duplication </A>
<UL TYPE=disc><LI><A HREF="http://www.cdoz.com.au">CD Oz</A>
 - CD-ROM <b>replication</b> facility.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/Countries/Canada/Provinces_and_Territories/Ontario/Counties_and_Regions/Peel/Cities/Mississauga/Business_and_Shopping/Business_to_Business/Computers/Software/Databases/">Regional > Countries > Canada > Provinces and Territories > Ontario > Counties and Regions > Peel > Cities > Mississauga > Business and Shopping > Business to Business > Computers > Software > Databases </A>
<UL TYPE=disc><LI><A HREF="http://www.PeerDirect.com/">PeerDirect</A>
 - offers heterogeneous database <b>replication</b> and synchronization across servers, desktops, laptops, and hand-held devices.</UL><p>
<A HREF="http://dir.yahoo.com/Regional/U_S__States/Virginia/Cities/Manassas/Business_and_Shopping/Business_to_Business/Computers/Software/">Regional > U.S. States > Virginia > Cities > Manassas > Business and Shopping > Business to Business > Computers > Software </A>
<UL TYPE=disc><LI><A HREF="http://www.datadisc.com/">dataDisc CD-ROM Tools and Technology</A>
 - provides mastering, <b>replication</b>, CDR, and document publishing services; sells CD software, recorders, blank discs, jukeboxes and  complete turnkey services.  </UL><p>
<A HREF="http://dir.yahoo.com/Regional/Countries/Canada/Business_and_Economy/Companies/Computers/Software/Databases/Titles/">Regional > Countries > Canada > Business and Economy > Companies > Computers > Software > Databases > Titles </A>
<UL TYPE=disc><LI><A HREF="http://www.PeerDirect.com/">PeerDirect</A>
 - offers heterogeneous database <b>replication</b> and synchronization across servers, desktops, laptops, and hand-held devices.</UL><p>

<center>
</center>

<p><center><table bgcolor="black" border=0 cellpadding=0 cellspacing=0><tr><td><table border=0 cellpadding=5 cellspacing=1><tr bgcolor="ccccaa"><td><b>
<a href="/search?&n=84&p=replication&&hc=0&hc=0&hs=173&h=s&b=105">Next 20 Matches</a>
</b></td></tr></table></td></tr></table></center>
<p>
<table bgcolor="#000000" border=0 cellpadding=0 cellspacing=0 width="100%"><tr><td><table border=0 cellpadding=2 cellspacing=1 width="100%"><tr align=center bgcolor="#eeeeee">
<td width="20%"><font size=-1 face=arial>
Categories
</font></td>
<td bgcolor="#a0b8c8" width="20%"><font size=-1 face=arial><b>Web Sites</b></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/bottom/*http://ink.yahoo.com/bin/query?p=replication&hc=0&hs=173">Web Pages</a></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/bottom/*http://search.news.yahoo.com/search/news?&n=84&p=replication&&hc=0&hc=0&hs=173">Related News</a></font></td>
<td width="20%"><font size=-1 face=arial><a href="http://rd.yahoo.com/search/navbar/bottom/*http://search.events.yahoo.com/search/events?p=replication&hc=0&hs=173">Net Events</a></font></td></tr></table></td></tr></table>
<center><form method=get action="/search">
<input size=35 name=p value="replication"><font size=-1 face=arial><input type=submit value="Next Search"></font>
<small><a href="/search/options?&n=84&p=replication&&hc=0">advanced search</a> - <a href="http://help.yahoo.com/help/search/">help</a></small><hr>
<small><B>Other Search Engines</b><br><a href="http://rd.yahoo.com/results/a/?http://www.altavista.digital.com/cgi-bin/query?pg=q&what=web&q=replication&user=yahoo">Alta Vista</a> - <a href="http://rd.yahoo.com/results/a/?http://www.goto.com/d/search/p/yahoo/?Keywords=replication">GoTo.com</a>  - <a href="http://www.yahoo.com/results/a/?http://www.wired.com/cgi-bin/redirect/1393/http://www.hotbot.com/?SM=MC&MT=replication&DV=25&RG=.com&DC=10&DE=2&OPs=MDRTP&_v=2&DU=days&SW=web">HotBot</a> - <a href="http://rd.yahoo.com/results/a/?http://www.infoseek.com/Titles?qt=replication&svx=yahoo">Infoseek</a> - <a href="http://rd.yahoo.com/results/a/?http://www.deja.com/[ST_cam=search.yahoo.none.slot]/dnquery.xp?QRY=replication&DBS=2&maxhits=20&format=terse&showsort=score&ST=QS&LNG=ALL">Deja.com</a> - <a href="http://dir.yahoo.com/Computers_and_Internet/Internet/World_Wide_Web/Searching_the_Web/">More...</a><p>
<a href="http://yp.yahoo.com/">Yellow Pages</a> - <a href="http://people.yahoo.com/">People Search</a> - <a href="http://maps.yahoo.com/">City Maps</a> - <a href="http://dir.yahoo.com/Regional/U_S__States/">Get Local</a> - <a href="http://events.yahoo.com/">Today's Web Events &amp; Chats</a> - <a href="http://isurf.yahoo.com/cgi-bin/y/keyword_search.cgi?db=/data/global_keyword&q=replication">Image Surfer</a> - <a href="http://docs.yahoo.com/docs/family/more.html"><b>More Yahoos</b></a><hr>
<em>Copyright &copy; 1994-1999 Yahoo! Inc. All Rights Reserved.</em> - <a href="http://docs.yahoo.com/docs/info/">Company Information</a> - <a href="http://docs.yahoo.com/docs/info/help.html">Help</a> - <a href="http://add.yahoo.com/fast/help/search/cgi_feedback?from_url=http://help.yahoo.com/help/search/search-09.html">Search Feedback</a> </small></form></center></body></html>
