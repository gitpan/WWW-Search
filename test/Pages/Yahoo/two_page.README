This set of test-result pages was created on Mon Aug 16 16:36:44 1999
