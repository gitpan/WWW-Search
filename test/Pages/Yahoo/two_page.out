http://www.replication.com
http://www.tapematic.com
http://www.replication.com
http://www.syware.com/datasync/datasync.htm
http://www.PeerDirect.com/
http://www.di.uminho.pt/~cbm/wmr96.html
http://www.mcs.net/~aaron/mememath.html
http://www.accessexcellence.org/AB/GG/influenza.html
http://www.MasterDisc.com
http://www.syware.com/datasync/datasync.htm
http://www.bio.umass.edu/biochem/simon/index.html
http://cpmcnet.columbia.edu/dept/luban_lab/
http://www.ags.com
http://www.goldenrom.com
http://www.cdpress.com
http://www.digitalforce.com
http://www.digital-oasis.co.uk
http://www.cmsduplication.com
http://www.esp-cd.com
http://www.failsafeinc.com
http://www.gatemusic.com
http://www.abbeytape.com/
http://www.merlinss.com/
http://www.mme.com.eg/
http://www.panasonicdvd.com/
http://www.saintmedia.com/
http://tachyon-tech.com
http://www.trace.com/
http://www.transitdigital.com
http://www.fischerartworks.com/
http://www.fsv.co.uk/
http://iccanet.com/
http://www.starbyte.com/starbyte.html
http://www.syndir.com/index2.html
http://www.techsonicscd.com
http://www.transmediatech.com
http://www.cdman.com
http://www.cdrinteractive.com
http://www.discmasters.com
http://www.duplisoft.com/
http://www.echodata.com/
http://www.image-src.com
http://www.gomrs.com
http://www.oasiscd.com/
http://www.rimage.com/
http://www.saicomm.com
http://www.saabconsulting.com
http://www.dbsg.com/
http://www.dbsg.com/
http://www.iti-pa.com
http://www.digitalbrothers.com
http://www.gatemusic.com
http://www.datamirror.com
http://www.linkpro.com/
http://www.castle.net/~knittuck/
http://good-morning.com/
http://www.shadow.net/~holodi/
http://www.vibrationresearch.com/
http://www.fischerartworks.com/
http://www.robinson-iron.com/
http://www.usod.com/
http://www.arrival-productions.com/
http://flosun.salk.edu/~forsburg/
http://www.bio.umass.edu/biochem/simon/index.html
http://www.4hometown.com
http://www.lionfox.com
http://www.discmasters.com
http://www.CDNDR.com/
http://www.1createamate.com/
http://www.zebra.net/~casper64/resume.html
http://www.erols.com/exie/linfeng.htm
http://flosun.salk.edu/~forsburg/
http://www.digitalsites.com
http://www.neonsoft.com/
http://www.octopustech.com/
http://www.maxdisk.com/
http://www.softpack.com/
http://www.softpack.com/
http://www.datadisc.com/
http://www.mburetta.com/
http://www.edp.com
http://www.scribesoft.com
http://www.edp.com
http://oak.cats.ohiou.edu/~biegalke/
http://www.gomrs.com
http://www.iti-pa.com
http://www.merlinss.com/
http://www.cdrinteractive.com
http://www.cdrom4u.com
http://www.cdoz.com.au
http://www.PeerDirect.com/
http://www.datadisc.com/
http://www.PeerDirect.com/
http://www.mme.com.eg/
http://www.biznet.com.my
http://www.wiredstudios.demon.co.uk
http://www.digital-oasis.co.uk
http://www.fsv.co.uk/
http://www.cdrom4u.com
http://www.cdpress.com
http://www.micrord.com/
http://www.cdpress.com
http://www.digital-oasis.co.uk
http://www.cdexperts.com
http://www.wiredstudios.demon.co.uk
http://www.erikarecords.com/
http://www.fsv.co.uk/
http://www.tiac.net/users/severino/
http://flosun.salk.edu/~forsburg/
http://www.trace.com/
http://www.fischerartworks.com/
http://www.fischerartworks.com/
http://www.discmasters.com
http://www.transmediatech.com
http://www.oasiscd.com/
http://www.cdman.com
http://www.saturn.ca
http://www.saintmedia.com/
http://www.genex.com
http://www.digivid.com/
http://www.vibrationresearch.com/
http://www.rimage.com/
http://www.scribesoft.com
http://www.esp-cd.com
http://cpmcnet.columbia.edu/dept/luban_lab/
http://www.echodata.com/
http://www.compactdisk.com
http://www.lionfox.com
http://www.cdman.com
http://www.mme.com.eg/
http://www.goldenrom.com
http://tachyon-tech.com
http://www.transitdigital.com
http://www.abbeytape.com/
http://www.saicomm.com
http://www.dbsg.com/
http://www.linkpro.com/
http://www.panasonicdvd.com/
http://www.digitalsites.com
http://www.syndir.com/index2.html
http://www.sangacorp.com/
http://www.CDNDR.com/
http://www.knr.net/
http://www.mburetta.com/
http://www.4hometown.com
http://www.bio.umass.edu/biochem/simon/index.html
http://www.knr.net/
http://www.image-src.com
http://www.robinson-iron.com/
http://www.digitalbrothers.com
http://www.ags.com
http://www.saabconsulting.com
http://www.SuperDD.com
http://www.failsafeinc.com
http://www.digitalforce.com
http://www.gilanet.com/amerabo
http://www.sentex.net/~bebop/
http://cust.iamerica.net/nativway/temp.htm
http://www.gilanet.com/amerabo
http://cust.iamerica.net/nativway/temp.htm
http://www.greatwar.com/replications/rmain.htm
http://www.greatwar.com/replications/rmain.htm
